src/routers/graph_memory.py

```
@@ -0,0 +1,655 @@
+        SELECT id, embedding <=> :query_embedding::vector AS distance
+        FROM documents
+        WHERE workspace_name = :workspace_name
+          AND collection_name = :collection_name
+          AND deleted_at IS NULL
+        ORDER BY embedding <=> :query_embedding::vector
+        LIMIT 5
+    """)
+    
+    # For now, use a simplified anchor search since we don't have the query embedding
+    # In production, the query is embedded by the Deriver's embedding client
+    anchor_result = await db.execute(
```

Ruff (0.15.18)

[error] Undefined name `db`

(F821)